app/renderer/components/task_suite.js

```diff
@@ -66,6 +67,10 @@ module.exports = {
         onDragStart() {
             this.event.emit("collapseTask");
         },
+        onAdd(evt){
```

Comment:
I think onAdd may be a misleading name (just below there is an `addTask` method completely unrelated)

Maybe `onDragAdd` `onTaskMovedEnd` or something like this would be better? It would be confusing to have an onAdd method that it is not called when adding a task.

On the other hand, because this logic is common to the add process and the onMoveEnd process, it could be useful to have the same method for both